### PR TITLE
Milestone desktop version

### PR DESCRIPTION
**Pull request by bertinkambale**
For the fourth project in my portfolio website, I will:

- [ ] I  Used media queries to make sure that my website displays correctly on desktop screens.
- 
- [ ] General requirements
- [ ] Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- [ ]  Make sure that you used correct ([Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- [ ]  Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,6 +38,11 @@
...
             />
           </svg>
+          <ul>
+            <li><a class="a-link">Portfolio</a></li>
+            <li><a class="a-link">About</a></li>
+            <li><a class="a-link">Contact</a></li>
+          </ul>
         </nav>
       </header>
       <section class="main-header container">
```
Comment (christianbiring1) [inline index.html:48]: - [ ] Kindly make sure you have the background images setup as you can see on the tempate.
image
Your design :point_down: 
image

Comment (christianbiring1) [reply index.html:48]: - [ ] Kindly make sure you use the correct typo `fonts` as the template. `hint: You can click on the text in figma and on the right you will be able to see the correct font and color`

File: index.html [hunk 2]
```diff
@@ -63,53 +68,63 @@ <h3>LET’S CONNECT</h3>
...
-        </p>
-        <ul class="ancrotag">
-          <li>html</li>
-          <li>css</li>
-          <li>javaScript</li>
-        </ul>
-        <div class="see-project4">
-          <button>See project</button>
+        <div>
+          <h1>Tonic</h1>
```
Comment (christianbiring1) [inline index.html:74]: - [ ] Kindly make sure there's only one header `h1` on the page. other headers can be `h2...6`

File: index.html [hunk 3]
```diff
@@ -63,53 +68,63 @@ <h3>LET’S CONNECT</h3>
...
+         </p>
+         <ul class="ancrotag">
+           <li>html</li>
+           <li>css</li>
+           <li>javaScript</li>
+         </ul>
+         <div class="see-project4">
+           <button>See project</button>
+         </div>
         </div>
```
Comment (christianbiring1) [inline index.html:121]: - [ ] Kindly make sure you use enough `padding` and `margin` to make enough space on from the edge.
Your design :point_down: 
image
Figma design :point_down: 
image

- [ ] Also consider making your images a little bit more responsive.

File: style.css [hunk 4]
```diff
@@ -44,16 +66,6 @@ body {
...
-  font-style: normal;
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  color: #6070ff;
-}
-
 .my-first-section h1 {
```
Comment (christianbiring1) [inline style.css:69]: - [ ] Kindly make sure your project is responsive as you can see the way your page is overflowing on the right when it comes to the mobile version
image

File: style.css [hunk 5]
```diff
@@ -493,3 +506,516 @@ body {
...
+    top: 0;
+    left: 0;
+    background-color: white;
+    width: 100%;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    padding-left: 60px;
+    padding-right: 60px;
+  }
+
```
Comment (christianbiring1) [inline style.css:524]: - [ ] Kindly make sure your page is responsive on all the screen size
image

File: style.css (hunk 4 above)
Comment (christianbiring1) [reply style.css:69]: image

File: style.css [hunk 6]
```diff
@@ -493,3 +506,516 @@ body {
...
+    left: 0;
+    top: 0;
+    padding: 15px 38px 15px 16px;
+    resize: none;
+    border-radius: 8px;
+  }
+
+  .form-control {
+    height: 48px;
+    width: 327px;
```
Comment (christianbiring1) [inline style.css:995]: - [ ] Kindly make sure you stick to the image as possible as you can.
image
Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @bertinkambale , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle:  
almost


## Highlights
- [ ] All linters checks are green
- [ ] Descriptive PR `tittle`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly makes sure you document your project in a professional way as well.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 7]
```diff
@@ -493,3 +513,516 @@ body {
...
+  .theSecondContainer h2 {
+    padding: 10px 0 30px 0;
+  }
+
+  .my-contact img {
+    padding-bottom: 10px;
+    padding-left: 350px;
+    display: flex;
+    justify-content: center;
+  }
```
Comment (emmiiorji) [inline style.css:982]: You have done a great job at centring the get in touch button as requested by the previous reviewer. Kindly ensure that the size of the button and the text alignment match as well. Also notice that the black bar does not appear on the desktop view.

Your design:
image


Figma design:
image

File: style.css [hunk 8]
```diff
@@ -493,3 +513,516 @@ body {
...
+    width: 100%;
+  }
+
+  .card-section {
+    display: grid;
+    gap: 50px;
+    flex-direction: column;
+    align-items: center;
+  }
+
```
Comment (emmiiorji) [inline style.css:625]: As suggested by the previous reviewer, kindly ensure that you maintain the margins around the project cards as given on your chosen template design.

Your design:
image


Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @bertinkambale, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost there

 ## Good Highlights✔

- Some requested changes have been implemented
- Linter checks are passing
- Good commit messages
- Adherence to GitHub flow


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- In order to reduce cluttering on the project's home directory, I suggest that you organize all the project images into a single directory.
    
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -63,53 +68,63 @@ <h3>LET’S CONNECT</h3>
...
```
Comment (Tadesse-Alemayehu) [inline index.html:123]: - [ ] You are doing great, addressing the requested changes. Also, as the previous reviewer mention, you should consider adding padding and margins to your projects, so they will look like the one provided on Figma. This is part of the main project requirement.

## Your design 
Screenshot from 2022-09-30 11-01-45

## Figma design 
Screenshot from 2022-09-30 11-02-39

**Review by Tadesse-Alemayehu — CHANGES_REQUESTED** [review]
Hi @bertinkambale 

You have donned incredible job on making the changes requested by a previous reviewer 👏 
But there are still some which aren't addressed yet.

request_change

 ## Highlights
- Most of the changes are implemented well. :heavy_check_mark: 
 - All linters checks are green  :heavy_check_mark: 
 - Descriptive PR tittle  :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**